### PR TITLE
add CSP sources for obsolete browser prompt

### PR DESCRIPTION
**Pull request by seve**
Adds script hash and explicit domain to `img-src` directive

### CODE REVIEW

File: server/eb/app.py [hunk 1]
```diff
@@ -96,12 +96,16 @@ def __init__(self, app_config):
...
+        obsolete_browser_script_hash = 'SHA25-0028D52E332C015C3ED9929926F4000BB4020B8CB85C1F5769D6AA3BA711F58E'
         csp = {
             "default-src": ["'self'"],
             "connect-src": ["'self'"],
-            "script-src": ["'self'", "'unsafe-eval'", "'unsafe-inline'"] + script_hashes,
+            "script-src": ["'self'", "'unsafe-eval'", "'unsafe-inline'"]
+            + obsolete_browser_script_hash + script_hashes,
             "style-src": ["'self'", "'unsafe-inline'"],
-            "img-src": ["'self'", "data:"],
+            "img-src": ["'self'", "data:", "https:///cellxgene.cziscience.com"],
```
Comment (seve) [inline server/eb/app.py:108]: Does order matter here? Should the domain be going before `"data:"`

Comment (tihuan) [reply server/eb/app.py:108]: That's a great question! I think the order shouldn't matter and the doc doesn't seem to mention it either 🤔 But definitely worth a double check lol 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/img-src

File: client/configuration/webpack/obsoleteHTMLTemplate.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <script>
+  /** Reflect ANY changes to this script in the script hash in `server/eb/app.py` **/
```
Comment (tihuan) [inline client/configuration/webpack/obsoleteHTMLTemplate.html:2]: Thanks for the comment!

File: server/eb/app.py (hunk 1 above)
Comment (seve) [inline server/eb/app.py:108]:
```suggestion
            "img-src": ["'self'", "data:", "'https:///cellxgene.cziscience.com'"],
```

Forgot single quotes!

File: server/eb/app.py [hunk 3]
```diff
@@ -105,7 +105,7 @@ def _before_adding_routes(app, app_config):
             "script-src": ["'self'", "'unsafe-eval'", "'unsafe-inline'"]
             + obsolete_browser_script_hash + script_hashes,
             "style-src": ["'self'", "'unsafe-inline'"],
-            "img-src": ["'self'", "data:", "https:///cellxgene.cziscience.com"],
+            "img-src": ["'self'", "data:", "'https:///cellxgene.cziscience.com'"],
```
Comment (tihuan) [inline server/eb/app.py:108]: Ahh so tricky 🤦 Sorry for missing that too! Do we need single quotes for `data:`?

Comment (seve) [reply server/eb/app.py:108]: Unit tests are still failing, so I guess, not sure why they weren't failing earlier ¯\_(ツ)_/¯

This also could be an issue of the source order.


```suggestion
            "img-src": ["'self'", "'data:'", "'https:///cellxgene.cziscience.com'"],
```

Comment (tihuan) [reply server/eb/app.py]: Hmm yeah and `data:` doesn't seem to need quotes 🤔 

https://stackoverflow.com/questions/40360109/content-security-policy-img-src-self-data

File: server/eb/app.py [hunk 4]
```diff
@@ -96,12 +96,16 @@ def __init__(self, app_config):
...
+        obsolete_browser_script_hash = 'SHA25-0028D52E332C015C3ED9929926F4000BB4020B8CB85C1F5769D6AA3BA711F58E'
         csp = {
             "default-src": ["'self'"],
             "connect-src": ["'self'"],
-            "script-src": ["'self'", "'unsafe-eval'", "'unsafe-inline'"] + script_hashes,
+            "script-src": ["'self'", "'unsafe-eval'", "'unsafe-inline'"]
+            + obsolete_browser_script_hash + script_hashes,
             "style-src": ["'self'", "'unsafe-inline'"],
-            "img-src": ["'self'", "data:"],
+            "img-src": ["'self'", "'data:'", "'https:///cellxgene.cziscience.com'"],
```
Comment (seve) [inline server/eb/app.py:108]:
```suggestion
            "img-src": ["'self'",  "'https:///cellxgene.cziscience.com'", "data:"],
```

**Pull request merged**